tests/test_sessions.py

```diff
@@ -186,3 +189,69 @@ def test_download_in_session(self, httpbin):
                  httpbin.url + '/get', env=self.env())
         finally:
             os.chdir(cwd)
+
+
+class TestExpiredCookie(SessionTestBase):
```

Comment:
Plural "cookies" would be better. Also, it doesn’t need to inherit from `SessionTestBase` (which would make sense to be renamed to `MultiStepSessionTestBase`).

Reply:
plural "cookies" definitely is better. However, the inheritance from SessionTestBase seems necessary due to the fact that the test involves creating a session file

Reply:
It may involve duplicating the logic in the `SessionTestBase.teardown_method(self,method)` function, as I'd have to clean up the session file created during the test